components/Controls.tsx

```diff
@@ -9,11 +9,29 @@ import { compareVersions } from 'compare-versions'
 
 export interface BikeControlsArgs {
     bike: Bike
-    api: Api
+    api: Api | undefined
     disconnect: () => void
 }
 
 export default function BikeControls({ bike, api, disconnect }: BikeControlsArgs) {
+    // Check if api is available
+    if (!api) {
+        console.error('Api not available')
```

Comment:
This code path is not an error thus this console.error can be removed

Reply:
Was for debugging but I forgot to remove it